src/binding/python/MeshRecordComponent.cpp

```diff
@@ -52,5 +53,40 @@ void init_MeshRecordComponent(py::module &m) {
             &MeshRecordComponent::setPosition<long double>,
             "Relative position of the component on an element (node/cell/voxel) of the mesh")
 
+        .def(py::pickle(
+            [](const MeshRecordComponent &a) { // __getstate__
+                // Return a tuple that fully encodes the state of the object
+                Attributable::MyPath const myPath = a.myPath();
+                std::cout << myPath.filePath() << std::endl;
+                for( auto const & s : myPath.openPMDGroup )
+                    std::cout << s << std::endl;
+                return py::make_tuple(myPath.filePath(), myPath.openPMDGroup);
+            },
+            [](py::tuple t) { // __setstate__
+                if (t.size() != 2)
+                    throw std::runtime_error("Invalid state!");
+
+                std::string const filename = t[0].cast< std::string >();
+                std::vector< std::string > const openPMDGroup =
+                        t[1].cast< std::vector< std::string > >();
+
+                // Create a new openPMD Series and keep it alive forevaaa
+                // TODO: how much of a hack is that, heh? :D omg.
+                static auto series = openPMD::Series(
```

Comment:
Not sure how much of a hack this `static` member is to fix up lifetimes, but it seems to work :)